compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1209,4 +1209,16 @@ object messages {
            |${parents.mkString("  - ", "\n  - ", "")}
            |""".stripMargin
   }
+
+  case class VarArgsParamMustComeLast()(implicit ctx: Context)
+    extends Message(IncorrectRepeatedParameterSyntaxID) {
+    override def msg: String = "*-parameter must come last"
```

Comment:
let's say `"varargs parameter must come last"`, mixing symbols and text like this feels kind of weird IMO